internal/local/app.go

```diff
@@ -236,6 +238,33 @@ func (a *App) LoadConfig() error {
 	return nil
 }
 
+// ConfigContainsVersion returns true if the app root config file contains a version
+func (a *App) ConfigContainsVersion() bool {
```

Comment:
what if we tried to limit ourselves to only reading that file once?

initially, the intended difference b/w `FindApp` and `LoadApp` was supposed to be that "find" would locate the app but not have to read any files/data.  obviously that distinction was a bit short-sighted, as this current bug indicated.  "FindApp" is still going to have to read data in order to truly find itself

in that sense, there's probably not much distinction needed b/w `LoadAppConfig` and `FindApp`

so instead of `FindApp` using the scanner to check for a matching config version, we should just skip right to reading the data and parsing json, to see that we have a non-zero value of `config_version`

Reply:
this is an interesting point! I did a bit of digging & I think its safe to moved the call to `LoadApp()` from `LoadAppConfig` to within `FindApp`